docs/user_guide/assessment.rst

```diff
@@ -216,6 +216,72 @@ case we see that the subgroup ``(a, 8)`` has a result of ``NaN``, indicating
 that there were no samples in it.
 
 
+.. _control_features_metrics:
+
+Control features for grouped metrics
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Control features (sometimes called 'conditional' features) also divide the input
+data into subgroups. The difference is that aggregations (such as
+:code:`MetricFrame.group_max`) are performed for each subgroup in the conditional
+feature(s), rather than across them (as happens with the sensitive features).
+
+To make this more concrete, consider a loan scenario. In addition to sensitive
+features such as gender and race, applicants can be placed into income bands.
+In this scenario, disparity based on income band is to be expected, but within
+each income band we would want to ensure that there is no disparity across the
+sensitive features. To do this, we make the income band into a control feature
+(in the statistical sense of 'controlling for a variable').
+
+The :class:`MetricFrame` constructor allows us to specify control features in
+a manner similar to sensitive features, using a :code:`conditional_features=`
+parameter:
+
+.. doctest:: assessment_metrics
+    :options:  +NORMALIZE_WHITESPACE
+
+    >>> income_band = ['l','l','h','l','h','l','l','h','h','l','l','h','h','l','l','h']
+    >>> metric_c_f = MetricFrame(skm.precision_score,
+    ...                          Y_true, Y_pred,
+    ...                          sensitive_features=g_2,
+    ...                          control_features=income_band)
+    >>> metric_c_f.overall
+    control_feature_0
+    h     0.8
+    l    0.25
+    Name: precision_score, dtype: object
+    >>> metric_c_f.by_group
+    control_feature_0  sensitive_feature_0
+    h                  6                      0.666667
+                       8                             1
+    l                  6                           0.5
+                       8                             0
+    Name: precision_score, dtype: object
+
+Note how the :attr:`MetricFrame.overall` property is stratified based on the
+supplied control feature. With the :class:`MetricFrame` computed, we can
+perform aggregations:
+
+.. doctest:: assessment_metrics
+    :options:  +NORMALIZE_WHITESPACE
+
+    >>> metric_c_f.group_max()
+    control_feature_0
+    h    1.0
+    l    0.5
+    Name: precision_score, dtype: float64
+    >>> metric_c_f.difference(method='between_groups')
```

Comment:
Adding a comment after the results of this command explaining in one line what they are seeing.

Reply:
Added a comment in the code and in the text.